src/ToolTip/ToolTip.js

```diff
@@ -0,0 +1,58 @@
+import Tooltip from '@mui/material/Tooltip';
+import { dcnb } from 'cnbuilder';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button } from '../Button/Button';
+import { alignOptions } from './ToolTip.levers';
+
+export const ToolTip = ({ alignment, className, children, text, ...props }) => {
+  const toolTipStyles = {
+    tooltip: dcnb(
+      className,
+      '!su-px-9 !su-py-6 !su-sans !su-text-14 !su-text-white !su-bg-cool-grey !su-color-cool-grey !su-shadow !su-max-w-[50ch]'
+    ),
+    tooltipArrow: '!su-bg-cool-grey !su-color-cool-grey',
+  };
+
+  return (
+    <Tooltip
+      {...props}
+      title={text}
+      placement={alignment}
+      classes={toolTipStyles}
+    >
+      <Button
+        variant="unset"
+        size="unset"
+        className="su-leading-trim su-shadow-none hocus:su-shadow-none su-rounded-full su-p-0 children:su-m-0 children:su-text-black hocus:children:su-text-digital-blue active:children:su-text-digital-blue"
+        icon="info"
+      />
+    </Tooltip>
+  );
+};
+
+// Prop Types.
+// -----------------------------------------------------------------------------
+ToolTip.propTypes = {
+  // Bubble alignment
+  alignment: PropTypes.oneOf(alignOptions),
+
+  // CSS Classes.
+  className: PropTypes.string,
+
+  // Children
+  children: PropTypes.oneOfType([
+    PropTypes.node,
+    PropTypes.element,
+    PropTypes.string,
+  ]),
```

Comment:
Just as an FYI, `PropTypes.element` and `PropTypes.string` are unnecessary here as they are [already encompassed by `PropTypes.node`](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes). The most accurate typing for `children` using `prop-types` would be the following:
```
children: PropTypes.oneOfType([
  PropTypes.node,
  PropTypes.arrayOf(PropTypes.node),
]),
```
The reasoning behind this being that sibling children are represented as ordered arrays. e.g.
```
<ParentComponent>
  <ChildA />
  <ChildB />
</ParentComponent>
// ParentComponent.props.children = [ChildA, ChildB]
```

Reply:
hmmm... It looks like `PropTypes.node` might even encompass arrays now as well, so you might be able to just use that. 🤔